TwitchPlaysAssembly/Src/Commands/ModuleCommands.cs

```diff
@@ -76,6 +76,8 @@ public static void BombTurnAroundCancel(TwitchModule module)
 
 	public static IEnumerator Show(TwitchModule module, object yield)
 	{
+		if (TwitchPlaysService.Instance.OnInteractCommand != null)
+			TwitchPlaysService.Instance.OnInteractCommand.Invoke();
```

Comment:
Would calling this from TwitchBomb.Focus handle all three cases you have? If so, I think that would be better.
Also you can write this as `TwitchPlaysService.Instance.OnInteractCommand?.Invoke();`

Reply:
Yes that would handle the cases from what I've tested. I believe I've put it in the right place but if it makes sense to move it else where in the method let me know or you can change it if you wish.